osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -169,7 +169,6 @@ private void load(BeatmapDatabase beatmaps, AudioManager audio, DialogOverlay di
 
             if (osu != null)
                 playMode.BindTo(osu.PlayMode);
-            playMode.ValueChanged += val => Beatmap.PreferredPlayMode = val;

```

Comment:
How could this be triggered prior to `LoadComplete` in the first place? External code shouldn't modify objects with in-progress async load in the first place.